docs/docs/tooling/protobuf.md

```diff
@@ -0,0 +1,135 @@
+---
+sidebar_position: 1
+---
+
+
+# Protocol Buffers
+
+It is known that cosmos-sdk uses protocol buffers extensively, this docuemnt is meant to provide a guide on how it is used in the cosmos-sdk. 
```

Comment:
```suggestion
It is known that Cosmos SDK uses protocol buffers extensively, this docuemnt is meant to provide a guide on how it is used in the cosmos-sdk. 
```

Throughout the docs we word it as `Cosmos SDK`, it thinks we should keep that. It (subjectively) looks nicer too :)